anoncreds/protocol/primary/primary_claim_issuer.py

```diff
@@ -73,7 +74,7 @@ def _genPrime(cls):
         return prime
 
     async def issuePrimaryClaim(self, schemaId: ID, attributes: Attribs,
-                                U) -> PrimaryClaim:
+                                U) -> (PrimaryClaim, Dict[str, Sequence[str]]):
```

Comment:
Is the value type really Sequence[str]? Looks like it's a sequence of Tuples, isn't it?